packages/adblocker/src/engine/engine.ts

```diff
@@ -776,10 +776,14 @@ export default class FilterEngine extends EventEmitter<EngineEventHandlers> {
         } else if (filter.isGenericHide() || filter.isSpecificHide()) {
           hideExceptions.push(filter);
         } else if (filter.isException()) {
-          exceptions.push(filter);
+          if (filter.isRemoveParam()) {
+            redirects.push(filter);
```

Comment:
Why do we store removeparam as redirect rules? Aren't they of different nature?

Reply:
They're different but removeparam is implemented using redirection.

Reply:
Outdated.